extensions/ql-vscode/src/query-results.ts

```diff
@@ -160,10 +161,12 @@ export async function interpretResultsSarif(
   sourceInfo?: cli.SourceInfo
 ): Promise<SarifInterpretationData> {
   const { resultsPath, interpretedResultsPath } = resultsPaths;
+  let res;
   if (await fs.pathExists(interpretedResultsPath)) {
-    return { ...JSON.parse(await fs.readFile(interpretedResultsPath, 'utf8')), t: 'SarifInterpretationData' };
+    res = await sarifParser(interpretedResultsPath);
```

Comment:
Just a thought....if this function throws an error because the file is not parseable, should we delete the file and re-run `cli.interpretBqrsSarif`?

Possibly, we can do this as a followup.

Reply:
Hm, that's a good point 👍 I'll leave this comment up for now, while we work out the last invalid SARIF test issue..